cmd/gen-manifests/main.go

```diff
@@ -363,6 +363,32 @@ func mockDepsolve(packageSets map[string][]rpmmd.PackageSet, repos []rpmmd.RepoC
 
 				specSet = append(specSet, spec)
 			}
+
+			// generate pseudo packages for the config of each transaction
+			var setRepoNames []string
+			for _, setRepo := range pkgSet.Repositories {
+				setRepo.Hash()
```

Comment:
Should we be doing something with the hash here?

Reply:
It calculates and sets the Id property

Reply:
Ugh, sorry, we actually don't need it.

I initially tested using the Id instead of the name but it was too long and harder to read.  Dropping the line.

Reply:
Also I was wrong, it doesn't set the Id, it just returns it. :upside_down_face: